app/views/user_mailer/invite_increase_notification.text.erb

```diff
@@ -1,9 +1,4 @@
 <% content_for :message do %>
-Hi, <%= @user.login %>!
-
-We just wanted to let you know that you have <%= @total %> new invitation(s), which can be used to create new accounts at the archive. You can invite a friend from your invitations page at <%= user_invitations_url(@user) %>.
-
-Cheers,
-<%= ArchiveConfig.APP_SHORT_NAME %>
+<%= t 'user_mailer.invite_increase_notification.txt', login: @user.login, total: @total, invitation_page:   user_invitations_url(@user) , archive_name: ArchiveConfig.APP_SHORT_NAME %>
```

Comment:
There appear to be several spaces after "invitation_page:". One is enough.
Also, space goes *after* a comma, never  before. i.e.: the comma should be straight after "user_invitations_url(@user)", there shouldn't be a space there.